docs/configuration.md

```diff
@@ -25,7 +25,7 @@ Check out the [example configuration files](https://github.com/shinesolutions/ae
 | Name | Description | Required? | Default |
 |------|-------------|-----------|---------|
 | aem.profile | AEM Profile, check out the [list of available profiles](https://github.com/shinesolutions/puppet-aem-curator/blob/master/docs/aem-profiles-artifacts.md) | Optional | `aem62_sp1_cfp9` |
-| aem.keystore_password_parameter | Parameter store object which contains the [Java Keystore](https://www.digitalocean.com/community/tutorials/java-keytool-essentials-working-with-java-keystores) password used in AEM Author and Publish. | Mandatory | |
+| aem.keystore_password_parameter | Parameter store object which contains the [Java Keystore](https://www.digitalocean.com/community/tutorials/java-keytool-essentials-working-with-java-keystores) password used in AEM Author and Publish. Each / represents a subkey in Hiera. This is expected to exist under the root "/" so you must ommit it from the start. For example aem-opencloud/stack_prefix/aem-keystore-password is what goes in the config but becomes /aem-opencloud/stack_prefix/aem-keystore-password on consumption. Note: The Parameter name cannot contain dots/periods. | Mandatory | |
```

Comment:
The leading slash is still supported, so we shouldn't add the note to omit it. We don't have to explain the internal works regarding the code consumption as that can/will change throughout future versions. Regarding how hiera works for slashes being subkeys, is there an external page that we can link to?

Reply:
Ill remove the internal workings part.

There is no wiki resources that explain the slashes, i believe this is specific to the SSM Module itself and not so much a hiera querk as it is a module one.

Reply:
Here's the resource that explains SSM parameter names https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-parameter-name-constraints.html

Reply:
Here's the explanation about Hiera dotted notation which explains why the parameter name can't have dot otherwise it would be misinterpreted by Hiera https://puppet.com/docs/puppet/5.4/hiera_automatic.html#hiera-dotted-notation .